### ISSUE TITLE
In ConstraintEdge, is there a way to find corresponding LLVM instruction?

### ISSUE THREAD

**Issue by charlesxsh** [SVF-tools/SVF#394]
I am trying to analyzing a LLVM IR file with 
```
Andersen pta(pag);
pta.analyze();
```
I know in PAG each edge/node can find corresponding LLVM value, is there a way for ConstraintEdge to find its corresponding LLVM instruction (I would like to see which LLVM instruction cause this constraint)?

**Comment by yuleisui** [comment]
The initial constraint graph is a copy of PAG. However, the constraint graph can add copy edges during pointer analysis, but PAG will be fixed. The LLVM instruction information can be obtained through the `toString()` method of each PAGEdge

**Comment by charlesxsh** [comment]
Hi Yulei, thanks for the response. So suppose I have a `ConstraintEdge`, I cannot find LLVM value like what we can do for the `PAGEdge.getValue()` method right?

**Comment by yuleisui** [comment]
No. This is because a constraint edge added (e.g., copy) during solving, may not correspond to an LLVM instruction.
